packages/stark-ui/src/modules/action-bar/components/action-bar.component.html

```diff
@@ -58,20 +52,20 @@
 			<mat-icon svgIcon="dots-vertical" class="stark-small-icon"></mat-icon>
 		</button>
 		<mat-menu #menu="matMenu" xPosition="before">
-			<div
-				*ngFor="let action of alternativeActions; trackBy: trackAction"
-				mat-menu-item
-				[id]="actionBarId + '-alt-' + action.id"
-				[disabled]="!action.isEnabled"
-				(click)="onClick(action, $event)"
-			>
-				<ng-container *ngIf="action.isVisible !== false">
-					<div [ngClass]="action.className">
-						<mat-icon [svgIcon]="action.icon" class="stark-small-icon"></mat-icon>
-						<span translate>{{ action.label }}</span>
-					</div>
-				</ng-container>
-			</div>
+			<ng-container *ngFor="let action of alternativeActions; trackBy: trackAction">
+				<button
+					*ngIf="action.isVisible !== false"
```

Comment:
Same here: is this intended? is it tested?

Reply:
Same here: https://github.com/NationalBankBelgium/stark/pull/1648#discussion_r381937580